avplayback.js

```diff
@@ -143,19 +148,31 @@ instance.prototype.action = function(action) {
 			break;
 
 		case 'playClip':
-			cmd = 'AVP|1|start,' + opt.clip;
+			cmd = 'AVP|1|PgmStart,' + opt.clip;
```

Comment:
so here, rather change it to for example:

```cmd = 'AVP|1|PgmStart,' + ('0' + opt.clip).substr(-2);```

which will make it 01, 02, etc, and also handle 10, 11 correctly.